### PR TITLE
[TS] LPS-130397 | Properly process DL references in page CSS

### PR DESCRIPTION
**Pull request by wanderlast**
Hi Echo team!

Currently if a client adds custom CSS that uses a document from the DL, the URL is not transformed upon export/import to other instances of Liferay. With this fix we use the existing DLReferences content processor to properly process these URLS upon export/import of a theme. Per PTR-2323, I've sent this to your team since it seemed to have been confirmed by you guys to be a bug, but please let me know if this should go elsewhere. Thanks!

### CODE REVIEW

**Comment by wanderlast** [comment]
ci:test:sf

**Comment by wanderlast** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/ThemeImporterImpl.java [hunk 1]
```diff
@@ -82,11 +81,9 @@ public void importTheme(
 			colorSchemeId = colorSchemeIdAttribute.getValue();
 		}
 
-		String css = GetterUtil.getString(headerElement.elementText("css"));
```
Comment (ealonso) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/ThemeImporterImpl.java:85]: Hey @wanderlast,
Why this change is necessary? Could you please confirm it with the Staging Team?
Regards,
Eudaldo

Comment (wanderlast) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/ThemeImporterImpl.java:85]: Hi @ealonso,

Thanks for looking at this! It's necessary because, without it, we're not actually taking the CSS that's been reprocessed for DLReferences. We're left with CSS that still has DLReferences in it (e.g. { Background-image: url("[$dl-reference=/group/39902/com.liferay.document.library.kernel.model.DLFileEntry/39908.xml$]"); }). However, I've contacted them to see if there's any possible issues with the change here: https://issues.liferay.com/browse/PTR-2405?focusedCommentId=2450023&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2450023

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
Hi @wanderlast, will leave this on hold until we know what the Staging Team says. Thanks!

**Comment by jkappler** [comment]
Hi @wanderlast, did you get any feedback from the Staging Team on this?

Thanks

**Comment by wanderlast** [comment]
Hi @jkappler,
They told me they're still looking into it.

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
Changing the label to Ready to Merge, after reading the comment in https://issues.liferay.com/browse/PTR-2405?focusedCommentId=2453315&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2453315, since it seems to be a legacy functionality.

cc: @wanderlast

**Comment by wanderlast** [comment]
ci:test:staging

**Comment by ealonso** [comment]
ci:forward
